### PR TITLE
QE: Add containerized proxy to 5.0 BV

### PR DESCRIPTION
**Pull request by nodeg**
This will add the new containerized proxy to the 5.0 BV. I had to remove some of the options, since they are not available on the containerized proxy. See https://github.com/uyuni-project/sumaform/blob/90ef843cc7a62184328bfe23d13d4393c5957480/modules/proxy_containerized/main.tf

Depends on https://github.com/uyuni-project/sumaform/pull/1529

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf [hunk 1]
```diff
@@ -201,28 +200,24 @@ module "server_containerized" {
...
+  product_version    = "head"
+  name               = "pxy"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:a2"
+    memory             = 4096
+  }
+  server_configuration = {
+    hostname = "suma-bv-50-srv.mgr.suse.de"
+    username = "admin"
+    password = "admin"
```
Comment (szachovy) [inline terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:215]: Not a fan of such a things in publicly available projects, can we at least encrypt it somehow and decrypt in the moment of authorization?

Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:215]: This is not a public repository.
Only SUSE employees have access to it.

**Review by srbarrios — APPROVED** [review]

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf (hunk 1 above)
Comment (szachovy) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:215]: Are you sure? I was able to access this comment from Incognito mode being signed out.

Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:215]: > Not a fan of such a things in publicly available projects, can we at least encrypt it somehow and decrypt in the moment of authorization?
This was simply copy-pasted from the Uyuni/4.3 BV file.

**Pull request merged**